include/openXpsEnums.h

```diff
@@ -67,13 +67,13 @@ typedef enum xpHashField
     XP_TU_ETHER_C_TAG_DEI,
     XP_TU_ETHER_C_TAG_VID,
     XP_TU_ETHER_ETHERTYPE,
-    XP_RESERVED1,
-    XP_RESERVED2,
-    XP_RESERVED3,
-    XP_RESERVED4,
-    XP_RESERVED5,
-    XP_RESERVED6,
-    XP_RESERVED7,
+    XP_ULL_ETHERNET_MAC_DA,
```

Comment:
@pkarashchenko please refer to the openxps rules. remove and rebuild the openXPS binary.